src/tests/test_integration/test_profiles.py

```
@@ -0,0 +1,635 @@
+    result_profile = await db_session.execute(stmt_profile)
+    profile_in_db = result_profile.scalars().first()
+    assert profile_in_db is None, "Profile should not be created when S3 upload fails!"
+
+
+@pytest.mark.asyncio
+@pytest.mark.unit
+@pytest.mark.parametrize("first_name, last_name, expected_error", [
+    ("John1", "Doe", "John1 contains non-english letters"),
+    ("John", "Doe1", "Doe1 contains non-english letters"),
+])
+async def test_profile_creation_invalid_name(client, jwt_manager, first_name, last_name, expected_error):
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)